test/setupPolyfills.js

```diff
@@ -0,0 +1 @@
+require('raf').polyfill();
```

Comment:
React v16 now makes use of `requestAnimationFrame`, so we need to polyfill it in node environments.

Reply:
Is this a concern that should be left for apps like Chalice?